pages/login.tsx

```diff
@@ -1,35 +1,56 @@
 import * as React from 'react'
+import { Formik, Form, Field } from 'formik'
+import Input from '../components/Input'
+import { signupValidation } from '../helpers/formValidation'
```

Comment:
Don't you think that would be better to use its own validation like `loginValidation` or else to rename it by `formValidation` to be more generic